### DISCUSSION TITLE
uploading multiple files from sveltekit to pocketbase.

### DISCUSSION THREAD

**Discussion by FrenchFryInteractive** [pocketbase/pocketbase#3184] [Q&A]
Can anyone give me some references on how to do this? I have been over the documentation, but i can only get one image uploaded, even though i have 4 images selected.

**Comment by ganigeorgiev** [comment]
The plain JS docs example shows multiple files upload - https://pocketbase.io/docs/files-handling/#uploading-files (notice the `for` loop and `formData.append` call).

If that doesn't help, please share your code sample.
